simulator/export/export.go

```diff
@@ -524,3 +578,18 @@ func (s *Service) applyPods(ctx context.Context, r *ResourcesForImport, eg *util
 func isSystemPriorityClass(name string) bool {
 	return strings.HasPrefix(name, "system-")
 }
+
+// isSystemNamespace returns whether the given name of Namespace is prefixed with `kube-` or not.
+// The `kube-` prefix is reserved by Kubernetes, and users cannot create a Namespace with such a name.
+// See: https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#working-with-namespaces
+//
+// So, we need to exclude these Namespaces when importing/exporting any Namespaces.
+func isSystemNamespace(name string) bool {
+	return strings.HasPrefix(name, "kube-")
+}
+
+// isIgnoreNamespace returns whether the given name of Namespace is ignored namespace or not.
+// It's system reserved one and default namespace.
+func isIgnoreNamespace(name string) bool {
+	return isSystemNamespace(name) || name == "default"
```

Comment:
Currently, we also ignore "default" namespace. This is because the "default" will be created automatically by Kubernetes.
But there is a possibility that users may be labeled to a "default" namespace and hope to import?
WDYT?